viewer/db.js

```diff
@@ -215,18 +223,21 @@ function fixIndex (index) {
     }).join(',');
   }
 
-  // If prefix isn't there, add it. But don't add it for sessions2 unless really set.
-  if (!index.startsWith(internals.prefix) && (!index.startsWith('sessions2') || internals.prefix !== 'arkime_')) {
-    index = internals.prefix + index;
-  }
+  // Don't fix extra  user-specified indexes from the queryExtraIndices
+  if (!Array.isArray(internals.info.queryExtraIndices) || !internals.info.queryExtraIndices.includes(index)) {
```

Comment:
this `fixIndex` function modifies index names given to it by adding prefixes/suffixes (e.g., `arkime_`, etc.). However the user-defined `queryExtraIndices` values should not be adjusted in this way.